_toc.yml

```diff
@@ -0,0 +1,18 @@
+# Table of contents
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: README
```

Comment:
This is the landing page. your README is nice, so I used that. You could make a .md or .ipynb you'd like to use instead.